src/components/searchResults/searchResults.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { hot } from "react-hot-loader";
+import "./searchResults.scss";
+import MovieList from "../movieList/movieList";
+
+
+class SearchResults extends React.PureComponent {
+    constructor (props) {
+        super(props);
+        this.state = {}; // is necessary to show result/no result part
+    }
+
+    render () {
+        return (
+            <div className="searchResults__wrapper">
+                <div className="container">
+                    <div className="searchResults">
```

Comment:
Move `no results` to a separate component, please

Reply:
Done